### ISSUE TITLE
bug: 

### ISSUE THREAD

**Issue by xSaoul** [favna/discordjs-docs-parser#203]
**Describe the bug**
Neither URL or Link properties will work due to Discordjs moving their docs quite a while ago. Unless I'm mistaken all urls should be replaced from
`https://discord.js.org/#/docs/discord.js/`
to
`https://old.discordjs.dev/#/docs/discord.js/`

**To Reproduce**

1. Search or get any doc
2. Use .url or .link
3. You will be shown a broken link to the homepage of Discord.js

**Expected behavior**
Be given a link to the exact page in the docs at `https://old.discordjs.dev/#/docs/discord.js/`

**Additional context**

This is an easy fix on the user end but personally I would rather avoid it. Again, I might be mistaken and using the package wrong but thought I'd ask/report in case its not the intended functionality.

Result of doc.link and doc.url on Client doc
```bash
[Client](https://discord.js.org/#/docs/discord.js/main/class/Client)
https://discord.js.org/#/docs/discord.js/main/class/Client
```

Expected
```bash
https://old.discordjs.dev/#/docs/discord.js/main/class/Client
```

**Comment by xSaoul** [comment]
Just wanted to also mention this issue persists through all sources.
`https://discord.js.org/#/docs/builders/`
Should be
`https://old.discordjs.dev/#/docs/builders/`

**Comment by favna** [comment]
This is a known issue and this library has actually gone into security-bug fixing only mode only I never ran a deprecate on npm yet. DiscordJS for their own helper bot (which is public btw, you can add it to your own server, just use `/invite` in the DJS server for `d.js docs` bot) which uses a *completely* different system for the builders, ws, etc packages and will move to the new style for the discord.js docs when its updated to match as well.

I wrote this lib because at the time the lib it was based on wasn't getting updated anymore but the code is honestly a total clusterfuck and I've already informed Souji that I do not intend to update it for the new docs. Once the old discord.js code has been updated to the new docs and the official bot is updated I will archive this repository.

The primary reason is that the new docs uses PlanetScale database for text search rather than relying on slow and clunky JSON parsing. There is just so, so much better about it that I could never match without being very active in the internals chat and get all the credentials and such.

TL;DR: Known issue, won't be fixed, invite the official bot to your server.
